bitmind/image_transforms.py

```diff
@@ -80,18 +82,20 @@ def __call__(self, img):
         return img
 
 
-# transforms to prepare an image for the base miner.
+# transforms to prepare an image for the base miner
 base_transforms = transforms.Compose([
     CenterCrop(),
-    transforms.Resize((224, 224)),
+    transforms.Resize(TARGET_IMAGE_SIZE),
     transforms.ToTensor(),
     transforms.Lambda(lambda t: t.expand(3, -1, -1) if t.shape[0] == 1 else t),
 ])
 
-# example data augmentation
+# data augmentation
 random_aug_transforms = ComposeWithParams([
-    RandomRotationWithParams(20),
-    RandomResizedCropWithParams(256, scale=(0.2, 1.0), ratio=(1.0, 1.0)),
+    transforms.ToTensor(),
+    transforms.Lambda(lambda t: t.expand(3, -1, -1) if t.shape[0] == 1 else t),
```

Comment:
can leave comment about using this for image type catching/handling (convert grayscale/1 channel images to 3 channel)

Reply:
Should we also handle images with 4 channels (alpha channel?)

Reply:
Good point - I realize now that the ImageDataset class handles alpha channels (prior to calling any transforms), but validator transforms don't. Replaced this lambda with `.convert('RGB')` to handle both L and RGBA